src/providers/Social/comment-sdk.ts

```diff
@@ -78,6 +79,7 @@ export async function createComment(
         );
         resolve(comment);
       } catch (error) {
+        Alert.alert('', error.message);
```

Comment:
Please discuss with your PM about this error message, in my opinion we should not use error from BE directly because it too technical

Reply:
BE return error message "This contains block word" if user comment or post with 'NFSW' word. 


And PM wants to show error alert.

Reply:
Search specific keyword and only alert for nfsw error return. Thanks